apps/nsqd/nsqd.go

```diff
@@ -182,12 +201,9 @@ func main() {
 
 	if flagSet.Lookup("version").Value.(flag.Getter).Get().(bool) {
 		fmt.Println(version.String("nsqd"))
-		return
+		os.Exit(0)
 	}
 
-	signalChan := make(chan os.Signal, 1)
-	signal.Notify(signalChan, syscall.SIGINT, syscall.SIGTERM)
```

Comment:
It's fine for now, but perhaps users of `go-svc` want to specify what signals they want to handle?  Right now it's hardcoded https://github.com/judwhite/go-svc/blob/master/svc/svc_other.go#L19